tests/e2e/defaultvmgroup_test.go

```diff
@@ -57,38 +65,106 @@ var _ = Suite(&DefaultVMGroupTestSuite{})
 		"We are able to remove vm from the _DEFAULT tenant which is NOT allowed as per current spec."))
 
-	log.Printf("END: defaultvmgroup.TestRemoveDefaultTenantVMs")
+	misc.LogTestEnd(c.TestName())
 }
 
 // Test step:
 //  Replace a vm from the _DEFAULT Vmgroup
 func (s *DefaultVMGroupTestSuite) TestReplaceDefaultTenantVMs(c *C) {
-	log.Printf("START: defaultvmgroup.TestReplaceDefaultTenantVMs")
+	misc.LogTestStart(c.TestName())
 
-	out, err := admincli.ReplaceVMFromVMgroup(s.esxIP, con.DefaultVMgroup, s.hostName)
+	out, err := admincli.ReplaceVMFromVMgroup(s.config.EsxHost, admincliconst.DefaultVMgroup, s.config.DockerHostNames[1])
 	c.Assert(err, IsNil, Commentf(out))
 	c.Assert(strings.TrimRight(string(out), "\n"), Equals, ErrorMsg, Commentf("Unexpected Behavior: "+
 		"We are able to replace vm from the _DEFAULT tenant which is NOT allowed as per current spec."))
 
-	log.Printf("END: defaultvmgroup.TestReplaceDefaultTenantVMs")
+	misc.LogTestEnd(c.TestName())
+}
+
+// Verify volume creation after deleting _DEFAULT tenant
+// 1. Create a volume
+// 2. Verify volume from docker host and esx
+// 2. Delete default tenant
+// 3. Verify not able to create volume from docker host - Error: VM does not belong to any vmgroup
+// 4. Verify docker volume ls does not show any volumes
+// 5. Verify admin cli shows volume with VMGroup set as N/A
+// 6. Create _DEFAULT vmgroup with —default-datastore=_VM_DS
+// 7. Again create new volume - operation should succedd
+// 8. Verify volume from esx and docker host
+func (s *DefaultVMGroupTestSuite) TestDeleteDefaultVmgroupAndVerify(c *C) {
+	misc.LogTestStart(c.TestName())
+	s.volumeName1 = inputparams.GetUniqueVolumeName(defaultVG)
+	s.volumeName2 = inputparams.GetUniqueVolumeName(defaultVG)
+
+	// Create a volume
+	out, err := dockercli.CreateVolume(s.config.DockerHosts[0], s.volumeName1)
+	c.Assert(err, IsNil, Commentf(out))
+
+	// Check if volume was successfully created
+	isAvailable := verification.CheckVolumeAvailability(s.config.DockerHosts[0], s.volumeName1)
+	c.Assert(isAvailable, Equals, true, Commentf("Volume %s is not available after creation", s.volumeName1))
+
+	// Verify if volume exists in default vmgroup
+	isVolInVmgroup := admincli.CheckVolumeExistInVmgroup(s.config.EsxHost, s.volumeName1, admincliconst.DefaultVMgroup)
+	c.Assert(isVolInVmgroup, Equals, true, Commentf("Volume [%s] does not belong to vmgroup [%s]", s.volumeName1, admincliconst.DefaultVMgroup))
+
+	admincli.DeleteVMgroup(s.config.EsxHost, admincliconst.DefaultVMgroup)
+	// Verify default vmgroup does not exist
+	isVmgroupAvailable := admincli.CheckVmgroupAvailability(s.config.EsxHost, admincliconst.DefaultVMgroup)
+	c.Assert(isVmgroupAvailable, Equals, false, Commentf("Falied to delete vmgroup %s .", admincliconst.DefaultVMgroup))
+
+	// Create a volume - operation should fail as default vmgroup no longer exist
+	out, err = dockercli.CreateVolume(s.config.DockerHosts[0], s.volumeName2)
```

Comment:
Same steps are done for volume1 and then for volume 2. 

Do you mind wrapping them in a func.

If there are more incoming test in this suite, it would be good to do this steps to  do these steps as part of test setup/or not.

Feel free to make the call.